core/src/com/unciv/logic/GameInfo.kt

```diff
@@ -238,6 +238,16 @@ class GameInfo {
                 .forEach { it.addNotification("A new barbarian encampment has spawned!", tile.position, Color.RED) }
     }
 
+    fun notifyCityStatesOfBarbarianEncampmentCleared(killer: CivilizationInfo, target: TileInfo) {
+        for (cityState in getAliveCityStates())
+            cityState.questManager.campKillNotification(killer, target)
+    }
+
+    fun notifyCityStatesOfConquest(killer: CivilizationInfo, target: CivilizationInfo) {
+        for (cityState in getAliveCityStates())
+            cityState.questManager.cityStateKillNotification(killer, target)
+    }
+
```

Comment:
I don't think there's any good reason for these functions to be in GameInfo, since they deal with extremely niche circumstances. Put them whey they'll be called.